pkg/k8s/authz.go

```diff
@@ -104,9 +104,3 @@ func ServiceProfilesAccess(k8sClient kubernetes.Interface) error {
 
 	return errors.New("ServiceProfile CRD not found")
 }
-
-// ClusterAccess verifies whether k8sClient is authorized to access all pods in
-// all namespaces in the cluster.
-func ClusterAccess(k8sClient kubernetes.Interface) error {
-	return ResourceAuthz(k8sClient, "", "list", "", "", "pods", "")
-}
```

Comment:
If anyone thinks we still need that, let me know why that might be the case

Reply:
@zaharidichev  that got added in #2474 apparently to ensure the control plane has enough RBAC to deal with pods in all the namespaces. I didn't see that check being explicit elsewhere in any of the healthchecks, so I think it's better to leave this in.

Reply:
mhmmm... should we add a bool to enable,disable it instead of leaving it unconditionally. It kind of feels strange to have to ensure that we have rbacs for pods to the remote cluster when we really do not need them

Reply:
True, that should be disabled for this case.